desktop-images/contactbg.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="1440" height="846" viewBox="0 0 1440 846" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M0 846H1440V0H146C65.3664 0 0 65.3664 0 146V846Z" fill="#6070FF"/>
+</svg>
```

Comment:
- [ ] Please there is a break in your cards. check and fix them. (I checked them from 2 different screens)

![p1](https://user-images.githubusercontent.com/63972670/160200476-99026cf5-537b-4dc2-a31c-815a4dfdcebd.JPG)

![p4](https://user-images.githubusercontent.com/63972670/160200509-ac9fcb17-71a4-4db3-bea5-8c7622a22ced.JPG)